regression/test.pl

```diff
@@ -244,6 +266,8 @@ ($$$$)
   -K         known: run tests associated with known bugs
   -D <key=value> Define - replace \$key string with "value" string in
                  test descriptors
+  -I <tag>   run only tests that have the given secondary tag. Can be repeated.
+  -X <tag>   exclude tests that have the given secondary tag. Can be repeated.
```

Comment:
1. I think this commit (extending test.pl) could be moved to a separate PR
2. Actual change request: please extend the documentation that describes the syntax of test.desc files to cover the new tagging mechansim.

Reply:
Ignore the first one, but the second item still stands.

Reply:
Done in the other PR; will drop this commit here.